README.md

```diff
@@ -124,7 +124,7 @@ First create a different user with the role `network-admin`.
 
 ~~~
 config term
-  username puppetuser password puppet role network-admin
+  user puppetuser password puppet role network-admin
 end

```

Comment:
This needs to remain as `username` this is a CLI input for the device itself.

Reply:
reverted